TLM/TLM/UI/Helpers/ISerializableOptionBase.cs

```diff
@@ -0,0 +1,9 @@
+namespace TrafficManager.UI.Helpers {
+
+    //legacy load and save
+    public interface ILegacySerializableOption
+    {
+        public void Load(byte data); // TODO keep this for backward compatibality.
+        public byte Save(); // TODO: delete this once xml serialization is ready
```

Comment:
You can't use `public` here.

Reply:
OK I will change it to internal then @krzychu124  is that OK?

Reply:
I am not sure why your IDE doesn't throw an error there. The only valid identifier is `new` at that point or not at all.

Reply:
@krzychu124 why do I need new at point of definition? Are you commenting on the wrong file?

Reply:
> why do I need new at point of definition?

You don't need `new` at this point, either public. I said that `new` it's the only valid and allowed keyword of interface member declaration which you can put there if you need (example below).

```cs
public interface ITest: IOther {
  new void DoSomething(); //you have to use 'new' here because it hides IOther.DoSomething declaration
}
public interface IOther {
  void DoSomething(); //you shouldn't add any modifier here, because no access modifiers are allowed on interface member declarations.
}
```